starter-code/index.js

```diff
@@ -1,11 +1,48 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
+    constructor() {
+      this.items = [];
+      this.length = this.items.length;
+    }
+    add(item) {
+      this.items.push(item);
+      this.length++;
+      this.items.sort((a, b) => a - b);
+    }
+    get(pos) {
```

Comment:
tu as oublié de throw l'erreur : 
get(pos) {
    if (pos > this.length) {
      throw new Error("OutOfBounds");
    }
    return this.items[pos];
  }